src/content/docs/en/guides/dev-toolbar.mdx

```diff
@@ -1,15 +1,15 @@
 ---
 title: Dev toolbar
-description: A guide to using the developer toolbar in Astro
+description: A guide to using the dev toolbar in Astro
 i18nReady: true
 ---
 import RecipeLinks from "~/components/RecipeLinks.astro";
 
-While the dev server is running, Astro includes a development toolbar at the bottom of every page in your local browser preview.
+While the dev server is running, Astro includes a dev toolbar at the bottom of every page in your local browser preview.
 
 This toolbar includes a number of useful tools for debugging and inspecting your site during development and can be [extended with more dev toolbar apps](#extending-the-dev-toolbar) found in the integrations directory. You can even [build your own toolbar apps](/en/recipes/making-toolbar-apps/) using the [Dev Toolbar API](/en/reference/dev-toolbar-app-reference/)!
 
-This toolbar is enabled by default and appears when you hover over the bottom of the page. It is a development tool only and will not appear on your published site.
+This toolbar is enabled by default and appears when you hover over the bottom of the page. It is a dev tool only and will not appear on your published site.
```

Comment:
```suggestion
This toolbar is enabled by default and appears when you hover over the bottom of the page. It is a development tool only and will not appear on your published site.
```
This isn't naming the toolbar itself, and should be spelled out in full!

Reply:
Good catch. Thanks!